overview/solving_problems.rst

```diff
@@ -0,0 +1,249 @@
+model traditionally used in statistical mechanics or its computer-science equivalent,
+the :term:`QUBO`, where variables are binary 0 and 1.
+
+.. [#]
+    Given :math:`N` variables :math:`\pmb{s}=[s_1,...,s_N]` corresponding
+    to physical Ising spins :math:`s_i \in \{-1,+1\}`, configurations of an Ising model
+    have energy,
+
+    .. math::
+
+        E(\pmb{s}|\pmb{h},\pmb{J})  = \left\{ \sum_{i=1}^N h_i s_i + \sum_{i<j}^N J_{i,j} s_i s_j  \right\}
+
+    where :math:`h_i` are biases and :math:`J_{i,j}` couplings between spins.
+
+Ocean software can abstract away much of the mathematics and programming. At its heart
+is a binary quadratic model (BQM) class for handling the desired objective function. It helps
+formulate objective functions for some common types of optimization problems.
+It also provides an API to binary quadratic :term:`sampler`\ s, such as the D-Wave
+system and classical algorithms you can run on your computer, which find the
+low-energy states that constitute solutions to the problem.
+
+The following sections give an intuitive explanation of these two steps (the
+third may benefit some problems) of this problem-solving procedure; see the :ref:`gs`
+examples and system documentation for further description.
+
+1. :ref:`formulating` as a BQM.
+2. :ref:`submitting` low-energy states of the BQM to find solutions.
+3. :ref:`improving`, if needed, using advanced features.
+
+.. figure:: ../_static/SolutionOverview.png
+   :name: SolutionOverview
+   :alt: image
+   :align: center
+   :scale: 90 %
+
+   Solution steps: (1) formulate a problem that you know in "problem space" (a circuit
+   of Boolean gates, a graph, a network, etc) as a BQM, mathematically or using
+   Ocean functionality and (2) sample the BQM for solutions.
+
+.. _formulating:
+
+Formulate a Problem
+===================
+
+There are different ways of mapping between any problem space (chains of amino acids
+forming 3D structures of folded proteins, traffic in the streets of Beijing, circuits
+of binary gates) and a BQM to be solved by sampling with a D-Wave system or locally on
+your CPU/GPU. Here we provide an intuitive example.
+
+Consider the problem of determining outputs of a Boolean logic circuit. In problem space
+the circuit might be described with input and output voltages, equations of
+its electronic components (resistors, transistors, etc), logic symbols,
+multiple or an aggregated truth table, and so on. You can mathematically
+formulate a BQM---in different ways too, for example a BQM for each gate or one BQM for
+all the circuit's gates---or use Ocean's formulations of binary gates directly in your
+code.
+
+For example, as shown in the :ref:`not` example, a NOT gate represented symbolically as
+:math:`x_2 \Leftrightarrow \neg x_1` in problem space might be formulated
+mathematically as the following QUBO:
+
+.. math::
+
+    E(a_i, b_{i,j}; x_i) = -x_1 -x_2  + 2x_1x_2
```

Comment:
This equation does not make sense. I think you mean to say

`E(x) = -x_1 - x_2 + 2 x_1 x_2`

If you want to put a, b in E the order matters

`E(x ; a, b) = \sum_i a_i x_i + \sum_i, j b_i,j x_i x_j`

Reply:
Unless you are solving the reverse Ising problem in which case `E(a, b ; x)` makes sense.